CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
If the minimum CMake version upgrade is too much, it can be kept as 3.10 but the required policy from 3.15 could be set to `NEW`.

This could affect older Linux distributions that wouldn't have CMake 3.15